README.md

```diff
@@ -1,2 +1,133 @@
-# to-do-list-website
-"To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete. 
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 [To-do-list] <a name="about-project"></a>
+
+**[To-do-list]** is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client </summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript#">JavaScript</a></li>
+    <li><a href="https://webpack.js.org">Webpack</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[built using webpack]**
+- **[interactive list]**
+- **[Add and remove items]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://VaheStepanyan100.github.io/to-do-list-website)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+To get a local copy up and running, follow these steps.
+
+- In this repository click on the "code" button (usually highlighted green) above.
+- You can select between either HTTPS or SSH if you have an SSH key linked.
+- Click on the copy icon on the right side of the link provided to copy the link
+- Go to your command line and type "git clone" followed by the link you just copied and press enter
+- The repository should be available locally on your computer.
+- You can change directory into the repository using the 'cd' command: cd "repository name"
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone git@github.com:VaheStepanyan100/to-do-list-website.git
+```
+
+### Usage
+
+To run the project, execute the following command:
```

Comment:
I suggest you make this statement complete as this can lead to a bad user experience and give your readme a less professional look👍.